entrypoint.sh

```diff
@@ -2,51 +2,29 @@
 
 set -e
 
-start () {
-    # This path could be configurable
-    # It is not really necessary to make it configurable since there is no
-    # usecase to change the database path inside the docker container
-    if [ -e /media/data/fact_wt_mongodb/REINITIALIZE_DB ]; then
-        python3 /opt/FACT_core/src/init_database.py && \
-            rm /media/data/fact_wt_mongodb/REINITIALIZE_DB
-    fi
-
-    # TODO This only works when the radare server runs on the docker host
-    # Probably the start_fact_frontend.py should not start radare but rather
-    # let the user start the radare container (and use docker-compose)
-    radare2_host=$(/sbin/ip route|awk '/default/ { print $3 }')
-
-    # If the user provides a different configuration the patch will fail
-    sed "s/RADARE2_HOST/$radare2_host/" /opt/FACT_core/0002_main_cfg.patch.template | patch --forward /opt/FACT_core/src/config/main.cfg || true
-
-    exec /opt/FACT_core/start_all_installed_fact_components "$@"
-}
-
 case "$1" in
-    "start")
+    "start-backend")
+        # We can't use exec since it will crash somehow
         shift 1
-        start $@
+        /opt/FACT_core/start_fact_backend "$@"
+        exit $?
     ;;
-    "start-branch")
+    "start-frontend")
         shift 1
-        git --git-dir=/opt/FACT_core/.git fetch
-        git --git-dir=/opt/FACT_core/.git checkout $1
-        shift 1
-        start $@
+        /opt/FACT_core/start_fact_frontend "$@"
+        exit $?
+    ;;
+    "initialize-db")
+        exec /usr/bin/python3 /opt/FACT_core/src/init_postgres.py
     ;;
     "pull-containers")
         # We can't to this in the Dockerfile, because the docker socket is not shared to there
         exec /opt/FACT_core/src/install.py \
             --backend-docker-images \
             --frontend-docker-images
     ;;
-    "pytest")
-        shift 1
-        cd /opt/FACT_core
-        pytest $@
-    ;;
```

Comment:
Why was this removed? We still need to be able to run pytest if we want to have a CI build for FACT docker at some point

Reply:
A noted in the README in section Development of FACT\_core in FACT\_docker I suggest to change the entrypoint to `/bin/bash` or `/usr/bin/pytest` or something similar.

Also to my knowledge the dependencies of the tests don't allow to run the tests in a distributed setup.

Reply:
> Also to my knowledge the dependencies of the tests don't allow to run the tests in a distributed setup.

That is a valid point 😅 
AFAIK @dorpvom wanted to have a CI build so we should maybe think about a way we could accomplish that

Reply:
Yeah, but this is something that should be fixed in fact_core rather than by a hack in the docker image.
Also as the docker containers are primarily supposed to be used by end users (at least for now) this is not something that the average user of the container will notice.